streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -2284,19 +2287,21 @@ abstract class ZStream[-R, +E, +O](val process: ZManaged[R, Nothing, ZIO[R, Opti
   /**
    * Runs the sink on the stream to produce either the sink's result or an error.
    */
-  def run[R1 <: R, E1 >: E, B](sink: ZSink[R1, E1, O, B]): ZIO[R1, E1, B] =
+  def run[R1 <: R, O1 >: O, E1 >: E, B](sink: ZSink[R1, E1, O1, Any, B]): ZIO[R1, E1, B] =
```

Comment:
Do you still need `O1 >: O` if `L = Any`?

Reply:
removed `O1`, no errors